nav2_system_tests/src/costmap_filters/speed_global_params.yaml

```diff
@@ -0,0 +1,323 @@
+amcl:
+  ros__parameters:
+    use_sim_time: True
+    alpha1: 0.2
+    alpha2: 0.2
+    alpha3: 0.2
+    alpha4: 0.2
+    alpha5: 0.2
+    base_frame_id: "base_footprint"
+    beam_skip_distance: 0.5
+    beam_skip_error_threshold: 0.9
+    beam_skip_threshold: 0.3
+    do_beamskip: false
+    global_frame_id: "map"
+    lambda_short: 0.1
+    laser_likelihood_max_dist: 2.0
+    laser_max_range: 100.0
+    laser_min_range: -1.0
+    laser_model_type: "likelihood_field_prob"
```

Comment:
If you're going to run another costmap filter test set with a custom config file, lets use another model here to help cover the amcl smoke testing at the same time.

Reply:
Changed to `beam` laser type for speed filter tests in https://github.com/ros-planning/navigation2/commit/34e694dbee7af9f805d2a7038b292466e44dc104#diff-7db8b39442f8f1e3c3dbce8f3864e3b6665064aa259904198bcee89742da09c3L19